src/main.ts

```diff
@@ -14,6 +14,7 @@ async function main(): Promise<void> {
     } catch (error) {
         core.setFailed(getErrorMessage(error));
     }
+    core.warning("THIS ACTION IS DEPRECATED. Use azure/cli@v2 to replace this action. Find it here: https://github.com/Azure/cli");

```

Comment:
I'm wondering if we should move this to the beginning instead of at the end.

I think we should point users to our document instead of the CLI action.

THIS ACTION IS DEPRECATED. Follow this guide to import your configuration into Azure App Configuration using GitHub Actions: https://aka.ms/appconfig/githubactions

Reply:
On the placing of the warning, I don't really have a strong opinion on it, I saw github add their warnings at the end for example, node 16 warning appeared at the end. 
![image](https://github.com/user-attachments/assets/a23f8b0c-d916-4108-b0b0-980a89e0f994)
I have updated the string.